package.json

```diff
@@ -2,7 +2,7 @@
   "name": "amorphic",
   "description": "Front to back isomorphic framework for developing applications with node.js and mongoDB",
   "homepage": "https://github.com/selsamman/amorphic",
-  "version": "2.4.3",
+  "version": "2.4.4",
```

Comment:
needs associated package-lock change, otherwise good to go

Reply:
as I was pushing it up lol